types/sorted_vec_set_type.toml

```diff
@@ -14,19 +14,13 @@ void getSizeType(const %1%<T,Compare, Allocator, GrowthPolicy, Container> &conta
 
 func = """
 template <class T, class Compare, class Allocator, class GrowthPolicy, class Container>
-void getSizeType(const %1%<T,Compare, Allocator, GrowthPolicy, Container> &container, size_t& returnArg)
+void getSizeType(const %1%<T,Compare, Allocator, GrowthPolicy, Container> &containerAdapter, size_t& returnArg)
 {
-    SAVE_SIZE(sizeof(%1%<T,Compare, Allocator, GrowthPolicy, Container>));
+    SAVE_DATA((uintptr_t)&containerAdapter);
 
-    SAVE_DATA((uintptr_t)&container);
-    SAVE_DATA((uintptr_t)container.capacity());
-    SAVE_DATA((uintptr_t)container.size());
+    SAVE_SIZE(sizeof(%1%<T,Compare, Allocator, GrowthPolicy, Container>) - sizeof(Container));
```

Comment:
I think a comment explaining this line would be helpful to prevent head-scratching in the future